hmmlearn/_hmmc.pyx

```diff
@@ -1,12 +1,11 @@
 # cython: boundscheck=False, wraparound=False
 
 cimport cython
-cimport numpy as np
 from numpy.math cimport expl, logl, isinf, INFINITY
 
 import numpy as np
 
-ctypedef np.float64_t dtype_t
+ctypedef double dtype_t
```

Comment:
Yes. There was a mix in the code so I standardized on `double`. This was unnecessary and I can revert it if you'd like.